WordPress/gutenberg
# Packages: Extract viewport package

Part of #3955.

This PR moves `viewport` module to `packages` folder and converts it to npm package maintained by Lerna. It's mostly boils down to moving files. In addition, I had to add `package.json` file and a few small adjustments to the existing setup.

**There are _NO_ breaking changes**

I also added a few unrelated changes where I discovered that we didn't list dev dependencies in other packages.

**Testing instructions**

 - Make sure the tests pass without regression
 - Make sure that editor still works
